src/SFA.DAS.AssessorService.Web/ViewModels/Certificate/CertificateBaseViewModel.cs

```diff
@@ -20,6 +20,7 @@ public virtual void FromCertificate(Domain.Entities.Certificate cert)
             Level = CertificateData.StandardLevel;
             Uln = cert.Uln.ToString();
             IsPrivatelyFunded = cert.IsPrivatelyFunded;
```

Comment:
Why is this still here?

Reply:
This plus your comments on the GetContactPreviousAddress are all interlinked. I originally dropped it from the base view model, but then there was no way to supply it into the GetContactPreviousAddress method. Which behind the scenes queries into this:

```
            var certificateAddress = await (from certificateLog in _context.CertificateLogs
                join certificate in _context.Certificates on certificateLog.CertificateId equals certificate.Id
                where statuses.Contains(certificate.Status) && certificateLog.Username == userName
                                                            && certificate.IsPrivatelyFunded == isPrivatelyFunded
                let certificateData = JsonConvert.DeserializeObject<CertificateData>(certificate.CertificateData)
```

I want to delete the last restriction clause when searching for a certificate address, which would remove the items you've highlighted. I'm unsure of the impact this has.

Reply:
I've removed privately funded, looking further i don't think it makes any sense to keep it in here it doesn't add value.